src/App.tsx

```diff
@@ -1,10 +1,10 @@
-import { useState } from "react";
-import "./App.scss";
-import { Header } from "./components/Header/Header";
+import { SetStateAction, useState } from 'react';
+import './App.scss';
+import { Header } from './components/Header/Header';
 
-function App() {
-  const [searchEntry, setSearchEntry] = useState("");
-  const handleSubmit = (searchInput) => {
+function App(): any {
+  const [searchEntry, setSearchEntry] = useState('');
+  const handleSubmit = (searchInput: SetStateAction<string>): void => {
```

Comment:
Are you sure the type of the parameter is of type SetStateAction? Isn't it a `string`? Usually `SetStateAction` is not used explicitly, as it inherited from `useState` call.

Reply:
Yes, it should be string. Thank you